### PR TITLE
#631 Sporty hq case study

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**

### CODE REVIEW

File: src/pages/case-studies/sporty-hq.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            sizes(quality: 100) {
+                ...GatsbyImageSharpSizes_withWebp
+            }
+        }
+        dior: imageSharp(id: { regex: "/dior.png/" }) {
+            sizes(quality: 100) {
+                ...GatsbyImageSharpSizes_withWebp
+            }
+        }
+        freightable: imageSharp(id: { regex: "/booksley/freightable.png/" }) {
```
Comment (CodeMeNatalie) [inline src/pages/case-studies/sporty-hq.js]: change frieghtable to ie. flymble

File: src/routes/SportyHQ/SportyHQ.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+          <div className={c.subtitle}>All about the game</div>
+          <div className={c.text}>
+            SportyHQ is a web platform built to power sports facilities{greaterThanSmall ? <br /> : ' '}
+            and governing bodies, run competitions, manage{greaterThanSmall ? <br /> : ' '}
+            membership and so much more.
+          </div>
+        </div>
+        <div className={cn(c.fullCol, c.border)}>
+          <div className={c.image}>
+            <Img sizes={data.tvMockup.sizes} />
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.js]: add alt

File: src/routes/SportyHQ/SportyHQ.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        </div>
+      </div>
+
+      <div className={c.thirdSection}>
+        <div className={c.fullCol}>
+          <div className={c.title}>A brand makeover</div>
+          <div className={c.subtitle}>Branding</div>
+        </div>
+        <div className={cn(c.image, c.firstImage)}>
+          <Img sizes={data.branding.sizes} />
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.js]: add alt

File: src/routes/SportyHQ/SportyHQ.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        </div>
+        <div className={cn(c.image, c.secondImage)}>
+          <Img sizes={data.icons.sizes} />
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.js]: add alt

File: src/routes/SportyHQ/SportyHQ.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+      </div>
+
+      <div className={c.fourthSection}>
+        <div className={c.fullCol}>
+          <div className={c.title}>Custom imagery for<br />that custom feeling</div>
+          <div className={c.subtitle}>Illustrations</div>
+        </div>
+        <div className={c.fullCol}>
+          <div className={c.image}>
+            <Img sizes={data.illustrations.sizes} />
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.js]: add alt

File: src/routes/SportyHQ/SportyHQ.js [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+      </div>
+
+      <div className={c.moreProjectsWrapper}>
+        <BrowseMoreProjectsSection
+          title="More projects"
+          firstUrl={diorUrl()}
+          firstTitle="Dior"
+          firstSubtitle="Worldwide clients management"
+          firstSrc={data.dior.sizes}
+          secondTitle="Freightable"
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.js]: change freightable to ie. Flymble

File: src/routes/SportyHQ/SportyHQ.module.scss [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+      }
+    }
+
+    .text {
+      padding-bottom: 100px;
+    }
+  }
+
+  .border {
+    padding-bottom: 200px;
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.module.scss]: `@include pb(7);`

File: src/routes/SportyHQ/SportyHQ.module.scss [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+    }
+  }
+
+  .centered {
+    @include flex(row, center, center);
+  }
+}
+
+.thirdSection {
+  padding-top: 200px;
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.module.scss]: `@include pt(7);`

File: src/routes/SportyHQ/SportyHQ.module.scss [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+
+    .title {
+      padding-bottom: 21px;
+
+      @include media-breakpoint-down(md) {
+        @include p(0);
+      }
+    }
+
+    .subtitle {
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.module.scss]: you are declaring subtitle and title class for the second time here with the same values. Make it once :)

File: src/routes/SportyHQ/SportyHQ.module.scss [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+      padding-bottom: 71px;
+
+      @include media-breakpoint-down(md) {
+        @include pb(1);
+      }
+    }
+  }
+
+  .firstImage {
+    padding-bottom: 200px;
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.module.scss]: `@include pb(7);`

also at least first Image, thirdSection and border classes has the same values. Can you clean it?

File: src/routes/SportyHQ/SportyHQ.module.scss [hunk 11]
```diff
@@ -0,0 +1,249 @@
...
+.moreProjectsWrapper {
+  padding-top: 181px;
+
+  @include media-breakpoint-down(md) {
+    @include pt(3);
+  }
+}
+
+.centered {
+  @include flex(row, flex-start, center);
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.module.scss]: `@include flex($justify: center);`

File: src/pages/case-studies/sporty-hq.js (hunk 1 above)
Comment (szczepanbarszczowski) [reply src/pages/case-studies/sporty-hq.js]: +1

File: src/routes/SportyHQ/SportyHQ.js (hunk 2 above)
Comment (szczepanbarszczowski) [reply src/routes/SportyHQ/SportyHQ.js]: +1

File: src/routes/SportyHQ/SportyHQ.js (hunk 3 above)
Comment (szczepanbarszczowski) [reply src/routes/SportyHQ/SportyHQ.js]: +1

File: src/routes/SportyHQ/SportyHQ.js (hunk 4 above)
Comment (szczepanbarszczowski) [reply src/routes/SportyHQ/SportyHQ.js]: +1

File: src/routes/SportyHQ/SportyHQ.js (hunk 5 above)
Comment (szczepanbarszczowski) [reply src/routes/SportyHQ/SportyHQ.js]: +1

File: src/routes/SportyHQ/SportyHQ.module.scss (hunk 8 above)
Comment (szczepanbarszczowski) [reply src/routes/SportyHQ/SportyHQ.module.scss]: +1

File: src/routes/SportyHQ/SportyHQ.module.scss (hunk 11 above)
Comment (szczepanbarszczowski) [reply src/routes/SportyHQ/SportyHQ.module.scss]: It's not working same way.

Comment (CodeMeNatalie) [reply src/routes/SportyHQ/SportyHQ.module.scss]: Weird O.o

File: src/routes/SportyHQ/SportyHQ.js [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+
+import { Layout } from 'layouts/Layout'
+import { BrowseMoreProjectsSection, DescriptionSection, HeroSection } from 'components/CaseStudiesElements'
+import { diorUrl, flymbleUrl } from '../../utils/url'
+import c from './SportyHQ.module.scss'
+
+export const SportyHQComponent = pure(({ data, greaterThanSmall }) => (
+  <div>
+    <Helmet>
+      <title>Case studies of our clients | 7ninjas</title>
```
Comment (CodeMeNatalie) [inline src/routes/SportyHQ/SportyHQ.js]: Flymble | 7ninjas

**Pull request merged**